src/components/BotBuilder/BotBuilder.css

```diff
@@ -306,7 +306,12 @@ input[type=file]{
 		margin-bottom: 40px;
 	}
 }
-
+@media (max-width:580px){
```

Comment:
Take the max-width as 579px. When all the elements are aligned in a single vertical line.